README.md

```diff
@@ -35,21 +35,21 @@ Mesos, etc.
 Visit [istio.io](https://istio.io) for in-depth information about using Istio.
 
 ## Istio authors
-Istio is an open source project with an active development community. The project was started
+Istio is an open-source project with an active development community. The project was started
 by teams from Google and IBM, in partnership with the Envoy team at Lyft.
 
-## Community meeting
+## Community Meeting
 
 We have public, [recorded](https://www.youtube.com/channel/UC-zVlo1F3mUbExQ96fABWcQ), community meetings.
```

Comment:
I haven't seen a community meeting in a while. Are these still happening?

Reply:
Looking at the Community meeting playlist, https://www.youtube.com/playlist?list=PL7wB27eZmdfcgqrjdftqafbOIWR630X4t, I don't see anything more recent than 7 months ago. I believe the Steering committee controls that meeting and also approves the PRs here. 

I know your PR is for grammatical things, and this isn't really that, but I think this is a good question for the approvers. An approver should know the answer about this meeting. Should there be a change here for the lack of meeting? If the Steering Committee is not holding the meeting, then we should remove from this page.

Reply:
Those meetings are currently not happening, however there are discussions happening in the steering to get this up and running.

Reply:
Great!

Reply:
My comment is.